ios/engine/KMEI/KeymanEngine/resources/Keyman.bundle/Contents/Resources/keyboard.html

```diff
@@ -39,6 +39,9 @@
                 kmw['getOskHeight'] = getOskHeight;
                 kmw['getOskWidth'] = getOskWidth;
                 kmw['setActiveElement']('ta');
+                kmw.getKeyboardPath = function(filename, package) {
+                    return filename;
+                };
```

Comment:
For the record, I really don't like this implementation, even if it does work.

Out of curiosity, what would be some 'average' filepaths that would pass through here - for example, the actual path this PR uses to load the default keyboard?

Reply:
Something like `file:///foo/bar/baz/keyman/languages/european2-1.6.js' right now

Reply:
So this is a temporary patch?

Reply:
Not temporary. Just that later it'll be `file:///foo/bar/baz/keyman/packages/european2/european2-1.6.js`.

My preference would be for KMW to be agnostic to package IDs since the package ID is not used in KMW other than to resolve file paths. Right now, we have code to resolve file paths in both KMW and in the apps. And we have to make sure that they're both consistent. I think it's best to just have the apps take care of locating the resources, since they have to do it anyway, rather than trying to do it again in KMW.

Reply:
One thing I will require here for final implementation - let's not ignore this component of the kmwembedded.js function:  `+ "?v=" + (new Date()).getTime(); /*cache buster*/`.  The keyboards do function in an embedded browser, so in case of keyboard updates we do want to ensure we can disregard the cache.

Based on our meeting about this, the change above should be moved into kmwembedded.js for KMW.  Once that's done, I don't mind taking a look (or perhaps Darcy) to ensure the needed one-time change is made for Android as well.

Reply:
Looks like I'm taking both sides of that task.